src/app/modules/main/chat_section/chat_content/view.nim

```diff
@@ -128,13 +128,13 @@ QtObject:
   proc amIChatAdmin*(self: View): bool {.slot.} =
     return self.delegate.amIChatAdmin()
 
-  proc updateChatDetails*(self: View, name, description, emoji, color, icon: string, ignoreName: bool) =
-    if not ignoreName:
-      self.chatDetails.setName(name)
-    self.chatDetails.setDescription(description)
-    self.chatDetails.setEmoji(emoji)
-    self.chatDetails.setColor(color)
-    self.chatDetails.setIcon(icon)
+  proc updateChatDetails*(self: View, chatDto: ChatDto) =
+    if chatDto.chatType != ChatType.OneToOne:
+      self.chatDetails.setName(chatDto.name)
+      self.chatDetails.setIcon(chatDto.icon)
+    self.chatDetails.setDescription(chatDto.description)
+    self.chatDetails.setEmoji(chatDto.emoji)
+    self.chatDetails.setColor(chatDto.color)
```

Comment:
The change looks pretty straight forward (replacing multiple params with a single one)
What's unclear to me however, is how that becomes the fix you're demonstrating in your videos attached to the PR.

From what I can see you've really just changed the method signature,

Reply:
Oh, I see. Thanks for clarifying. I've missed that.